### ISSUE TITLE
cpp dependency problems (eg boost)

### ISSUE THREAD

**Issue by dpastoor** [ronkeizer/PKPDsim#8]
Will need to add checks of some sort to computer to check if has the necessary boost libs installed in order to run the cpp function.

```
Compiling simulation function...
file4785e9ac7d3.cpp:5:10: fatal error: 'boost/array.hpp' file not found
#include <boost/array.hpp>
         ^
1 error generated.
make: *** [file4785e9ac7d3.o] Error 1
```

**Comment by ronkeizer** [comment]
pls try again, added boost headers now as package

**Comment by dpastoor** [comment]
worked, but got this warning FYI

```
Compiling simulation function...
In file included from file49801b9045.cpp:8:
In file included from /usr/local/Cellar/r/3.1.2/R.framework/Versions/3.1/Resources/library/BH/include/boost/numeric/odeint.hpp:29:
In file included from /usr/local/Cellar/r/3.1.2/R.framework/Versions/3.1/Resources/library/BH/include/boost/numeric/odeint/stepper/runge_kutta4.hpp:27:
In file included from /usr/local/Cellar/r/3.1.2/R.framework/Versions/3.1/Resources/library/BH/include/boost/numeric/odeint/stepper/explicit_generic_rk.hpp:28:
In file included from /usr/local/Cellar/r/3.1.2/R.framework/Versions/3.1/Resources/library/BH/include/boost/numeric/odeint/stepper/detail/generic_rk_algorithm.hpp:33:
/usr/local/Cellar/r/3.1.2/R.framework/Versions/3.1/Resources/library/BH/include/boost/fusion/sequence.hpp:7:2: warning: 'FUSION_SEQUENCE_10022005_0559' is used as a header guard here, followed by #define of a different macro [-Wheader-guard]
#if !defined(FUSION_SEQUENCE_10022005_0559)
 ^~
/usr/local/Cellar/r/3.1.2/R.framework/Versions/3.1/Resources/library/BH/include/boost/fusion/sequence.hpp:8:9: note: 'FUSION_ITERATOR_10022005_0559' is defined here; did you mean 'FUSION_SEQUENCE_10022005_0559'?
#define FUSION_ITERATOR_10022005_0559
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        FUSION_SEQUENCE_10022005_0559
1 warning generated.
Simulating...
Compiling simulation function...
```

**Comment by ronkeizer** [comment]
I get indeed similar messages now on Mac, was running it first on Linux without any warnings. Probably some unimportant compiler warning, might disappear with future versions of boost. Will try to suppress them for now.
